test/fuzz/transformation_add_dead_break_test.cpp

```diff
@@ -21,7 +21,7 @@ namespace {
 
 TEST(TransformationAddDeadBreakTest, BreaksOutOfSimpleIf) {
   // For a simple if-then-else, checks that some dead break scenarios are
-  // possible, and sanity-checks that some illegal scenarios are indeed not
+  // possible, and coherence-checks that some illegal scenarios are indeed not
```

Comment:
coherence -> integrity


illegal -> invalid

This is one of my pet peeves.  Nobody is going to arrest you for doing the wrong thing! :-)

Reply:
Haha :)

Done.